common-test/src/main/java/org/alfresco/hxi_connector/common/test/docker/util/DockerContainers.java

```diff
@@ -213,7 +213,8 @@ public static WireMockContainer createWireMockContainer()
     public static WireMockContainer createWireMockContainerWithin(Network network)
     {
         return createWireMockContainer()
-                .withNetwork(network);
+                .withNetwork(network)
+                .withEnv("WIREMOCK_OPTIONS", "--global-response-templating --permitted-system-keys=AWS_.* --verbose");
```

Comment:
If you are not setting any `AWS_*` ENV VARs, than you can skip the: ` --permitted-system-keys=AWS_.*`

Reply:
done